### PR TITLE
chore: Update CODEOWNERS for @googleapis/cloud-storage-dpe

### PR DESCRIPTION
**Pull request by danielbankhead**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-storage-transfer/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -8,4 +8,4 @@
 *               @googleapis/yoshi-python
```
Comment (tseaver) [inline .github/CODEOWNERS:8]: Did you mean to add `@googleapis/cloud-storage-dpe` here as well?
```suggestion
*               @googleapis/yoshi-python @googleapis/cloud-storage-dpe
```

**Review by tseaver — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (danielbankhead) [reply .github/CODEOWNERS:8]: Ah yes, thank you

**Comment by tseaver** [comment]
@danielbankhead You'll have to flip the CLA label manually -- the bot sometimes b0rks merged suggestions.

**Comment by danielbankhead** [comment]
@googlebot I consent.

**Comment by danielbankhead** [comment]
@googlebot I fixed it.

**Pull request merged**